### PR TITLE
LOM6in1z: Add example with multiple forenames

### PR DESCRIPTION
**Pull request by mrwilson**
## Why

Inform pilot participants of additional testing records.

## What

Describe a sample passport with multiple forenames, as is permitted in the spec.

### CODE REVIEW

File: source/send-a-test-request-to-the-DCS.html.md.erb [hunk 1]
```diff
@@ -116,5 +116,37 @@ If your response does not look similar to this, [contact the DCS team](/support)
 
 Once you’ve received the data, you can [access the data in a DCS response](/access-the-data-in-a-DCS-response/#access-the-data-in-a-dcs-response).
 
+## Other test cases
```
Comment (PippaClarkGDS) [inline source/send-a-test-request-to-the-DCS.html.md.erb:119]:
```suggestion
## Use other test passport details
```

File: source/send-a-test-request-to-the-DCS.html.md.erb [hunk 2]
```diff
@@ -116,5 +116,37 @@ If your response does not look similar to this, [contact the DCS team](/support)
...
+
+We've prepared some additional sample passport details with specific characteristics for you to test against.
+
+### Multiple forenames
```
Comment (PippaClarkGDS) [inline source/send-a-test-request-to-the-DCS.html.md.erb:123]:
```suggestion
### Passport example with multiple forenames
```

**Review by PippaClarkGDS — COMMENTED** [review]
a couple of small tweaks Alex - let me know what you think. Will also have to send to tech writers for review before it is pushed live

File: source/send-a-test-request-to-the-DCS.html.md.erb [hunk 3]
```diff
@@ -116,5 +116,37 @@ If your response does not look similar to this, [contact the DCS team](/support)
 
 Once you’ve received the data, you can [access the data in a DCS response](/access-the-data-in-a-DCS-response/#access-the-data-in-a-dcs-response).
 
+## Use other test passport details
```
Comment (PippaClarkGDS) [inline source/send-a-test-request-to-the-DCS.html.md.erb:119]:
```suggestion
## Use other sample passport details
```

File: source/send-a-test-request-to-the-DCS.html.md.erb [hunk 4]
```diff
@@ -116,5 +116,37 @@ If your response does not look similar to this, [contact the DCS team](/support)
...
+
+We've prepared some additional sample passport details with specific characteristics for you to test against.
```
Comment (PippaClarkGDS) [inline source/send-a-test-request-to-the-DCS.html.md.erb:121]:
```suggestion
Here's some additional sample passport details with specific characteristics for you to test against.
```

**Review by PippaClarkGDS — COMMENTED** [review]
A couple of tiny tweaks/suggestions

**Review by PippaClarkGDS — APPROVED** [review]
Looks good - nice work, Alex

**Comment by mrwilson** [comment]
Thanks for your help @PippaClarkGDS !

**Pull request merged**